src/Components/App/App.test.js

```diff
@@ -1,9 +1,113 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import ReactDOM from 'react-dom';
+import { render, fireEvent, waitFor } from '@testing-library/react';
 import App from './App';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+import { getAllPokemon, getSinglePokemon } from '../../ApiCalls'
+import { singlePokemonMockData} from './Pokemon-Mock-Data';
+import { act } from 'react-dom/test-utils';
+// import '@testing-library/jest-dom/extend-expect';
+
+
+jest.mock("../../ApiCalls");
+
+describe('App', () => {
+
+  getAllPokemon.mockResolvedValue( [
+      {
+        "name": "bulbasaur",
+        "url": "https://pokeapi.co/api/v2/pokemon/1/"
+      },
+      {
+        "name": "ivysaur",
+        "url": "https://pokeapi.co/api/v2/pokemon/2/"
+      },
+      {
+        "name": "venusaur",
+        "url": "https://pokeapi.co/api/v2/pokemon/3/"
+      },
+      {
+        "name": "charmander",
+        "url": "https://pokeapi.co/api/v2/pokemon/4/"
+      },
+      {
+        "name": "charmeleon",
+        "url": "https://pokeapi.co/api/v2/pokemon/5/"
+  }])
+
+  getSinglePokemon.mockResolvedValue(singlePokemonMockData)
```

Comment:
Lives in another file, can easily just mock the 5 things we look at (height, weight, etc)